torch/_torch_docs.py

```diff
@@ -1752,7 +1752,8 @@ def merge_dicts(*dicts):
            r"""
 conj(input, *, out=None) -> Tensor
 
-Computes the element-wise conjugate of the given :attr:`input` tensor.
+Computes the element-wise conjugate of the given :attr:`input` tensor. If the input has a non-complex dtype,
+the output tensor shouldn't be mutated.
```

Comment:
as per @ezyang 's suggestion

Reply:
Actually, since we're exploring conjugate view, let me suggest we make an even stronger claim: we should tell users to flat out NOT mutate the output of this function, ever. If you avoid doing so, you will write code that is forwards compatible with any possible semantics of conj (it always returns a new copy, it returns a new copy sometimes, it always returns a view).

I despair about actually making sure people don't do this with just a doc update. We really need some way to mark views as non-mutable. Maybe this wouldn't even be that hard to do...